local-cli/runIOS/runIOS.js

```diff
@@ -27,15 +27,24 @@ function runIOS(argv, config) {
 }
 
 function _runIOS(argv, config, resolve, reject) {
-  const args = parseCommandLine([{
-    command: 'simulator',
-    description: 'Explicitly set simulator to use',
-    type: 'string',
-    required: false,
-    default: 'iPhone 6',
-  }], argv);
+  const args = parseCommandLine([
+    {
+      command: 'simulator',
+      description: 'Explicitly set simulator to use',
+      type: 'string',
+      required: false,
+      default: 'iPhone 6',
+    },
+    {
+      command: 'ios-project-path',
```

Comment:
I'm fine with `ios-project-path`, it makes it clear it's the path where the XCode project lives.

Regarding the "path" vs "dir" terminology - if you look at other places in the CLI, we mostly use "path" so let's use that.

We should also add this for Android, can you add that to this PR please or send a separate PR?
`react-native run-android --android-project-path Examples/Movies/`